app/models/est.rb

```diff
@@ -0,0 +1,35 @@
+class Est
+  include Indexable
+
+  self.settings = {
+    index: {
+      analysis: {
+        analyzer:
+            { custom_analyzer:        {
+              tokenizer: 'standard',
+              filter:    %w(standard asciifolding lowercase snowball) },
```

Comment:
It's worth thinking about for sure :+1: 

[asciifolding](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-asciifolding-tokenfilter.html) would allow our users to find a document with e.g. the name "René" by searching for "Rene", which is desired behavior (note that this is relevant to when a user does a search; the actual document content won't be altered by this filter).

The [snowball _filter_](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-snowball-tokenfilter.html) is a stemming algorithm that is language dependent. You can specify which language is being analyzed, so the most correct thing to do here would be to create a filter for each language, and analyze each field depending on which language it is in, like as shown in the snowball filter documentation. I recommend you do that (yes, there will be an analyzer for each language).